src/main/java/uk/gov/hmcts/reform/professionalapi/repository/PrdEnumRepository.java

```diff
@@ -11,4 +11,6 @@
 public interface PrdEnumRepository extends JpaRepository<PrdEnum, UUID> {
 
     List<PrdEnum> findByEnabled(String flag);
+
```

Comment:
normally we should not write any new code only for test

Reply:
This has been addressed in this commit: https://github.com/hmcts/rd-professional-api/pull/480/commits/dc4dc69e0df737dcc62fce07335d67d43f17c804